modules/v2/metadata.yaml

```diff
@@ -217,7 +217,7 @@ spec:
               source: github.com/GoogleCloudPlatform/terraform-google-alloy-db
               version: ">= 5.0.1"
             spec:
-              outputExpr: "{\"ALLOYDB_INSTANCE_HOST\" : primary_instance_ip}"
+              outputExpr: "{\"ALLOYDB_INSTANCE_HOST\" : primary_instance_ip, \"ALLOYDB_READ_REPLICAS\": env_vars.ALLOYDB_READ_REPLICAS}"
```

Comment:
Will this be comma separate list? Comma separated list is not supported in environment variable.